WalletWasabi.Fluent/ViewModels/WalletManagerViewModel.cs

```diff
@@ -98,6 +101,7 @@ public WalletManagerViewModel(WalletManager walletManager, UiConfig uiConfig)
 		public ReadOnlyObservableCollection<NavBarItemViewModel> Items => _items;
 
 		public WalletManager Model { get; }
+		public BitcoinStore BitcoinStore { get; }
```

Comment:
add a blank line between these. (same as 2 functions after the other)

Reply:
Done